RecoTracker/LSTCore/standalone/analysis/occupancy/compute_occupancies.ipynb

```diff
@@ -22,15 +22,7 @@
     "        # Each branch is added to the dictionary\n",
     "        for branch in branches:\n",
     "            all_branches[branch] = tree[branch].array(library=\"np\")\n",
-    "    return all_branches\n",
-    "\n",
-    "# Branches relevant to the occupancy selections\n",
-    "mod_occ_branches = ['module_layers', 'module_subdets', 'module_rings', 'module_eta',\n",
-    "                    'md_occupancies', 'sg_occupancies', 't3_occupancies', 't5_occupancies']\n",
-    "\n",
-    "# Root file generated with compile -d option turned on to generate relevant occupancy branches\n",
-    "file_path = \"occ_1000_p06.root\"\n",
```

Comment:
am I guessing correctly that the old derivation was based on 1000 events and the new on 500?
Considering some relatively large changes at 99.99% quantile, I suspect that this reduction in the occupancies may be chasing some tails of occasional combinatorial explosions. Are there always enough entries in the remaining 0.01% to make the decision? Something >>10 (e.g. > 30) would be good.

Reply:
Yes, in the slides Yonsi sent me it said she used a 500 event sample for the 0.8 occupancies, so I tried to match that here. I will check.